addons/compat_sog/subConfig/grad_trenches/config.cpp

```diff
@@ -0,0 +1,39 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        addonRootClass = QUOTE(COMPONENT);
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "loadorder_f_vietnam",
+            "grad_trenches"
+        };
+        skipWhenMissingDependencies = 1;
+        VERSION_CONFIG;
+    };
+};
+
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_Equipment {
+                class EGVAR(trenches,digEnvelopeSmall);
+                class GVAR(digSpiderhole): EGVAR(trenches,digEnvelopeSmall) {
+                    displayName = CSTRING(Action_DigSpiderhole);
+                    statement = QUOTE([ARR_2({_this call EFUNC(trenches,placeTrench)},[ARR_2(_this select 0,'vn_o_vc_spiderhole_01')])] call CBA_fnc_execNextFrame);
```

Comment:
we want these when grad isn't present

Reply:
I misread the original code then. I guess we can always add them in compat and `delete` in this subAddon 🤔

Reply:
handled in https://github.com/acemod/ACE3/pull/9204/commits/85db5170298092941264f25e7313badef7269d4b